test/e2e/e2e-test/kong/src/test/resources/expectedData.yaml

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+segmentItems:
+- serviceName: kong
+  segmentSize: eq 1
+  segments:
+  - segmentId: not null
+    spans:
+    - operationName: "/mock"
```

Comment:
I think you should verify another segment referencing this exit span?

Reply:
yeah

Reply:
We haven't to. You could set the traffic back to another point of Kong/Nginx, as a cycle. Then you have a way to check.

Reply:
I got it.